### PR TITLE
test: add e2e full test

### PR DESCRIPTION
**Pull request by SweeXordious**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Comment by evan-forbes** [comment]
same question here, is this blocked by #192?

**Comment by SweeXordious** [comment]
yes, updating the branch now

File: e2e/full_test.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestFullLongBehaviour mainly lets a multiple validator network run for 200 blocks, then checks if
+// the valsets and data commitments are relayed correctly.
+func TestFullLongBehaviour(t *testing.T) {
```
Comment (evan-forbes) [inline e2e/full_test.go:21]: the name "FullLong" is not very precise as to what is actually being tested, so I would definitely recommend using different name. 

Simply TestIntegrationTest or TestIntegrationSuite or anything standard

**Review by evan-forbes — APPROVED** [review]

File: e2e/full_test.go (hunk 1 above)
Comment (SweeXordious) [reply e2e/full_test.go:21]: What is does is run the orchestrator and relayer for  more than 200 blocks. It's a long test that runs the full behaviour :D 

`TestIntegrationTest` is too generic, at least the other name is stating `FullLong`... I will open an issue for it https://github.com/celestiaorg/orchestrator-relayer/issues/206

**Comment by SweeXordious** [comment]
@evan-forbes  Thanks a lot for the reviews :D

**Pull request merged**